src/ResourceManager/AzureBatch/Commands.Batch/Az.Batch.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.2.2'
+ModuleVersion = '0.3.0'
```

Comment:
@matthchr you can change this to `0.2.3` -- we plan on doing minor version bumps for Cloud Shell releases every two weeks (until we GA and release as `1.0.0`), so these one-off "preview" releases should do patch version bumps

Reply:
Okay cool, will fix.